### PR TITLE
Add fishtank sounds to cryo cell

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request

~~Note to maints, need to wait until:~~
- ~~#88462~~

~~Is merged before adding this.~~ It's merged now. We good to go.

Adds some fishtank sounds to cryo cell when it's in use.

https://github.com/user-attachments/assets/4b2ec114-f38b-4a56-b2ee-8b1f2b5289cd

https://github.com/user-attachments/assets/8b00179c-c4c3-4e4a-8165-94226c2fd5de

https://github.com/user-attachments/assets/c59b9eb9-7605-4548-8bca-5d9f716fdecc

https://github.com/user-attachments/assets/fd14e6d6-053e-4e17-bca8-c68b18f6c6d6

https://github.com/user-attachments/assets/4f6de5d6-3698-4c94-94a6-9f093c7f2e02

https://github.com/user-attachments/assets/ee238c51-0b84-4757-8b87-202aa89eeb37

https://github.com/user-attachments/assets/270babce-ad54-4dc6-a0f7-748c8c446f0f

https://github.com/user-attachments/assets/abde675a-c792-474c-9bfc-099e3d064bb2

https://github.com/user-attachments/assets/9f3f92d0-4acf-4a5d-b32b-4f9cee85c6b3

https://github.com/user-attachments/assets/c43058c1-844d-4698-97b7-24e2322b281f

Taken from - https://freesound.org/people/DudeAwesome/sounds/386023/
Converted into OGG and split into 5 second sections
License: CC BY 4.0

## Why It's Good For The Game
Human fishtank immersion.

## Changelog
:cl:
sound: Add fishtank looping sounds to cryo cells when in use. Sound is from https://freesound.org/people/DudeAwesome/sounds/386023/
/:cl:

### CODE REVIEW

File: sound/attributions.txt
Comment (Sadboysuss) [inline sound/attributions.txt:1]: would be nice if the attribution text was in the same folder as the files themselves
(`sound/machines/cryo`)

File: code/datums/looping_sounds/machinery_sounds.dm [hunk 1]
```diff
@@ -182,3 +182,19 @@
 	end_volume = 60
 	extra_range = MEDIUM_RANGE_SOUND_EXTRARANGE
 	falloff_exponent = 4
+
+/datum/looping_sound/cryo_cell
```
Comment (Sadboysuss) [inline code/datums/looping_sounds/machinery_sounds.dm:186]:
```php
/datum/looping_sound/proc/play(soundfile, volume_override)
	var/sound/sound_to_play = sound(soundfile)
	if(direct)
		sound_to_play.channel = sound_channel || SSsounds.random_available_channel()
		sound_to_play.volume = volume_override || volume //Use volume as fallback if theres no override
		SEND_SOUND(parent, sound_to_play)
	else
		playsound(
			parent,
			sound_to_play,
			volume,
			vary,
			extra_range,
			falloff_exponent = falloff_exponent,
			pressure_affected = pressure_affected,
			ignore_walls = ignore_walls,
			falloff_distance = falloff_distance,
			use_reverb = use_reverb
		)
```
`#define SOUND_RANGE 17`
`var/maxdistance = SOUND_RANGE + extrarange`
`listeners = get_hearers_in_view(maxdistance, turf_source)`

if you're not specifying the range for your looping sound, then it'll be the 17 (the default), I think 17 is pretty high.

**Review by grungussuss — COMMENTED** [review]
it's a good idea to check how it actually sounds ingame cause the loops might sound jarring when transitioning between eachother, shouldn't be too bad though

File: code/datums/looping_sounds/machinery_sounds.dm (hunk 1 above)
Comment (timothymtorres) [reply code/datums/looping_sounds/machinery_sounds.dm:186]: You aren't factoring in the sound falloff exponent and distance. Even if those aren't set it still gets applied.

```
	for(var/mob/listening_mob in listeners | SSmobs.dead_players_by_zlevel[source_z])//observers always hear through walls
		if(get_dist(listening_mob, turf_source) <= maxdistance)
			listening_mob.playsound_local(turf_source, soundin, vol, vary, frequency, falloff_exponent, channel, pressure_affected, S, maxdistance, falloff_distance, 1, use_reverb)
			. += listening_mob
```

Code_sZTquqcAJo

Falloff_disance is 6 and falloff_exponent is 1.

But on debugging there are a lot of sounds that are getting sent to mobs that shouldn't be within hearing distance due to the falloff. It's basically playing sounds with no volume... which is baaaaaaad.

File: sound/attributions.txt
Comment (timothymtorres) [reply sound/attributions.txt:1]: This is kind of overkill. Majority of sounds are placed in a large attribution file. Even the code/art license is one file.

**Pull request merged**